tests/contract_testcases/evm/selector_override.sol

```diff
@@ -1,8 +1,14 @@
 contract selector {
-	constructor() selector=hex"abcd" {}
-	modifier m() selector=hex"" {_;}
-	receive() payable external selector=hex"1" {}
-	fallback() external selector=hex"abc" {}
-	function i() internal selector = hex"ab_dd" {}
-	function p() private selector = hex"ab_dd" {}
+	@selector([0xab,0xcd])
+	constructor() {}
+	@selector([x])
+	modifier m() {_;}
+	@selector([1])
+	receive() payable external {}
+	@selector([0xabc])
+	fallback() external {}
+	@selector([0xab,0xdd])
+	function i() internal {}
+	@selector([0xab,0xdd])
```

Comment:
```suggestion
	@selector([0xab, 0xdd])
```

Reply:
It's a test case, it really does not matter